neptunelib/experiment.py

```diff
@@ -114,6 +114,7 @@ def get_numeric_channels_values(self, *channel_names):
         The returned DataFrame may contain NaNs if one of the channels has more values than others.
 
         :param channel_names: Names of the channels to retrieve values for.
+        :param aligh_x: Bool whether to join all `x_` channels into one `x` channel.
```

Comment:
@jakubczakon This implementation has a side effect: depending on what arguments I choose (align_on_x True or False) I can get different result (dataframe shape and definition of cols). I think that this can be confusing to the Users.

Reply:
I will update the docstrings before merging to dev. Thanks for pointing that out.

I think, for a lot of use cases this default actually makes a lot of sense. Conversely, it is hard to think of situations where someone would actually want to download the data the way it is now with both batch and epoch channels in one df.

I think there are 3 options now:
* have this parameter and the user can choose which option she wants
* leave this function as is but suggest to the users that they should use the `align_to_x` helper function
* Make the align_on_x=True the only option and suggest to the users that they should know what they download say:

```python
batch_channels = project.get_numerical_value_channels('batch_loss1','batch_loss2')
epoch_channels = project.get_numerical_value_channels('epoch_loss1','epoch_loss2')
```

Which option would you go for @kamil-kaczmarek @lukasz-walkiewicz @pitercl ?

Reply:
Giving it a bit of thought I decided to go with option 3 for simplicity.
Also, the current implementation doesn't work with python 2.7 so it would be an added bonus.